internal/slicer/slicer.go

```diff
@@ -60,17 +58,18 @@ func Run(options *RunOptions) (*Report, error) {
 		syscall.Umask(oldUmask)
 	}()
 
-	release := options.Selection.Release
 	targetDir := filepath.Clean(options.TargetDir)
-	targetDirAbs := targetDir
```

Comment:
[Note for reviewer]: There was an edge case caught by spread where the relative path where causing the report to misbehave and not add the mutated files. I have changed the test with relative paths in `slicer_test.go` to fail (regression test).

This happened because `targetDir` was used inconsistently together with `targetDirAbs`, that's why I have changed it so all the code in slicer uses the absolute version and there are no edge cases that we have to test separately.